.github/workflows/release-50_publish-docker.yml

```diff
@@ -181,32 +194,47 @@ jobs:
         run: |
           . ./.github/scripts/common/lib.sh
 
-          RELEASE_ID=$(check_release_id "${{ inputs.release_id }}")
-          release=release-$RELEASE_ID && \
+          release="${{ needs.validate-inputs.outputs.stable_tag }}" && \
           echo "release=${release}" >> $GITHUB_OUTPUT
 
           commit=$(git rev-parse --short HEAD) && \
           echo "commit=${commit}" >> $GITHUB_OUTPUT
 
-          tag=$(git name-rev --tags --name-only $(git rev-parse HEAD)) && \
-          [ "${tag}" != "undefined" ] && echo "tag=${tag}" >> $GITHUB_OUTPUT || \
-          echo "No tag, doing without"
+          tag="${{ needs.validate-inputs.outputs.version }}" && \
+          echo "tag=${tag}" >> $GITHUB_OUTPUT
```

Comment:
dq: what is the point of separating this via `&&`

as i understand this won't skip setting output if the tag is empty
<img width="233" alt="image" src="https://github.com/user-attachments/assets/a6ddbd1b-4af9-433d-b45c-de99bec153dd">

if this is not validation on empty, then it could've been just

```bash
echo "tag=${{ needs.validate-inputs.outputs.version }}" >> $GITHUB_OUTPUT
```

or it wouldn't work?

or

Reply:
Yep, you are right, I think it is a leftover from the past, when the validation was done here. Now the validation is done in the separate step, and it can be assigned directly as you suggest.

Reply:
done